plugins/ccgx/ccgx.quirk

```diff
@@ -69,20 +70,33 @@ Plugin = ccgx
 GType = FuCcgxDmcDevice
 Summary = Dock Management Controller Device
 ParentGuid = USB\VID_03F0&PID_096B
+Vendor = HP
 Name = HP USB-C/A Universal Dock G2
 CcgxDmcTriggerCode = 0x01
 CcgxImageKind = dmc-composite
 InstallDuration = 180
 RemoveDelay = 162000
 
+# HP Thunderbolt Dock G4 Root Hub
+[USB\VID_1D5C&PID_5801]
+Summary = USB Hub
+Name = Thunderbolt Dock G4 top most USB Hub
```

Comment:
Not sure I like "top most" here; maybe "parent" instead?

Reply:
In G4 Dock, USB Connections are below. 
  USB\VID_1D5C&PID_5801(USB HUB) -- USB\VID_03F0&PID_2488 (USB HUB) -- USB\VID_03F0&PID_0488 (G4 Dock DMC)
I think that it is better HP to choose the name. How about it?
But I can't add HP Yasir as a reviewer.

Reply:
If you get me the github account name of Yasir I can add him as a reviewer.

Reply:
Ignore me; i see syabbashp below. I've invited him to this repo.

Reply:
Thanks a lot.

Reply:
@hughsie 
 Has Yasir been added as a reviewer? I can't see Yasir on the reviewer list.

Reply:
Yasir has to accept the request first:

![Screenshot 2022-04-28 at 12-33-40 Manage access · fwupd_fwupd](https://user-images.githubusercontent.com/151380/165743386-c377a60b-c909-40d9-a4a7-ee68766bb9d5.png)

Reply:
Thanks for your checking.
I have sent an email to Yasir to check this request.

Reply:
IMO this should be `Thunderbolt Dock G4 USB hub`.  The actual logic of which is which can be observed by the topology.

Reply:
I see that this is the Fresco Logic USB hub device (which is part of the HP Thunderbolt G4 Dock) appearing in the list. To avoid confusion with perhaps we can replace Thunderbolt Dock G4 USB hub with just "Dock USB hub". Is that acceptable?